services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -1255,6 +1255,12 @@ static void emitPushJobStatusMetrics(
             pushJobStatusStats.recordBatchPushSuccessSensor();
           }
         }
+        // Append job duration in ms and seconds to log message
+        logMessage.append(", duration: ")
```

Comment:
@m-nagarajan I believe the regex for the push job details dashboard will need to be modified, since it ends in `$` and this extends the log message.

Reply:
We can modify the regex in the dashboard once this is shipped. All good from me 👌

Reply:
I want to change this to have per colo information anyways, so it's going to break regardless.